### PR TITLE
fraud notebook

### PR DESCRIPTION
**Pull request by hhbyyh**

### CODE REVIEW

**Comment by jason-dai** [comment]
@yiheng @shane-huang please take a look.

**Comment by yiheng** [comment]
It contains some ssd code, maybe remove them? And please resolve the conflict

**Comment by yiheng** [comment]
why checkin a jar file?

File: pipeline/fraudDetection/src/BigDLKaggleFraud.scala [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+object BigDLKaggleFraud {
+
+  def main(args: Array[String]): Unit = {
+    Logger.getLogger("org").setLevel(Level.ERROR)
+    Logger.getLogger("breeze").setLevel(Level.ERROR)
+    val conf = Engine.createSparkConf()
+      .set("spark.task.maxFailures", "1")
```
Comment (yiheng) [inline pipeline/fraudDetection/src/BigDLKaggleFraud.scala]: remove this?

File: pipeline/fraudDetection/src/BigDLKaggleFraud.scala [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    val spark = SparkSession.builder().master("local[1]").appName("ss").config(conf).getOrCreate()
```
Comment (yiheng) [inline pipeline/fraudDetection/src/BigDLKaggleFraud.scala]: appName "BigDL Fraud Detection Example"?

**Comment by yiheng** [comment]
Should we add some instructions guide people to run your notebook?

**Comment by hhbyyh** [comment]
Thanks for the review @yiheng . Added a readme.md

Moving ssd notebook is necessary as originally it was put directly in the notebook folder. If necessary, I can separate a PR for that.

**Comment by yiheng** [comment]
LGTM

**Comment by hhbyyh** [comment]
Thanks for the review.
also cc @SeaOfOcean for the folder structure change.

**Pull request merged**